_posts/2016-07-04-imam-week1.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title:  "CrowdCensus Team's Data Exploration"
+date:   2016-07-04
+author: Imam and Myeong
+---
+
+The goal of CrowdSensing Census project in DSSG 2016 is to develop a reliable and general model that can complement census data by making use of other real-time data such as OpenStreetMap data or Cellphone Detail Records (CDR) to determine the social economic conditions in a particular region. This work is meaningful for allowing data freshness and cost efficiency compared to census data, since government-initiated socio-economic statistics are expensive and often outdated (10-year term). 
+
+Specifically, a prediction model would be developed from available datasets targeting the city of Milano and the province of Trento, Italy. We target these cities as the Big Data Challenge initiative made it possible to open Telecom Italia data (CDR) to the public <a href="http://www.nature.com/articles/sdata201555" target="_blank">(Barlacchi, et al., 2015)</a>. For the ground truths, we are trying to make use of the census data that include, but are not limited to, the statistics about illiteracy, school degree, foreigners, rented dwelling, unemployment and work force. We first conducted Principle Component Analysis (PCA) for the non-monetary variables about citizen well-beings. The PCA was used to see whether the variables represent a single dimension of socio-economic status.
```

Comment:
"Specifically, a prediction model would..." => "Specifically, we are developing a prediction model"

"For the ground truths" => "As ground truth estimates of census-level variables, we are ..."

"foreigners" => "proportion of foreigners in a census tract"

"work force" => "work force participation"

Maybe you want to link to the Wikipedia article on PCA?

"well-beings" => "well-being"